spec/models/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it "has attributes" do
+    user = User.find_or_create_by_auth({'info' => {uid: 1, name: "fake name", image: "fakeurl.com/image.gif", email: 'example@gmail.com'}})
```

Comment:
Useless assignment to variable - `user`.<br>Line is too long. [139/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.